### ISSUE TITLE
VS Code UI. Add option to remove VS Code logo from the Title bar.

### ISSUE THREAD

**Issue by YehorZXC** [microsoft/vscode#209944]
By default, Visual Studio Code displays its logo in the top left corner of the title bar.
As a user I would like to change its appearance to my custom logo or hide it completely.

The logo is part of the title bar and there is currently no setting in VS Code that allows for its removal or replacement.
This is a design choice made by the VS Code team and cannot be changed through the settings.

That logo does not seems to be an important UI element.
Double-clicking it maximizes or restores the window, but that functionality already exists with a single click on the middle window control button. Since the logo is not essential, it would be great if we could hide it from the title bar, similar to how other UI elements like the menu bar, command center, and layout controls can be hidden.

Personally I would hide that VS Code logo leaving more space to the menu bar or replace it with the hamburger menu.
I think giving users the option to customize the logo would be a great improvement.

To sum up I would like to have an option to hide/customize VS Code logo in the title bar. Because it duplicates maximize/restore functionality of the default window control button(maximize that on the middle) and it is not customizable yet.
I look forward to hearing your thoughts on this proposal. I believe this feature would be a valuable addition to VS Code.

**Comment by babakfp** [comment]
You can use this app (on Windows) to add your custom styles to the vsCode UI. There are vsCode extensions that do the same.

Add this style to remove the logo:

```css
.monaco-workbench .part.titlebar>.titlebar-container>.titlebar-left {
    display: none;
}
```

The content in the center of the title bar will be moved to the right side.

Add this style to hide the logo:

```css
.monaco-workbench .part.titlebar>.titlebar-container>.titlebar-left {
    opacity: 0;
}
```

The center content in the center of the title bar will stay in the center.

---

To Open dev tools, press `ctrl` + `shift` + `p` and select `Developer: Toggle Developer Tools`.
